16/umbraco-cms/fundamentals/setup/upgrading/README.md

```diff
@@ -1,11 +1,15 @@
 ---
-description: This is the guide for upgrading existing Umbraco installations.
+description: Select the guide that fits your upgrade situation and learn more about the recommended approach.
 ---
 
 # Upgrade your project
 
 In this section, you will find everything you need to upgrade your Umbraco CMS project.
 
-If you are new to upgrades, see our introduction to [upgrades in Umbraco](./upgrade-introduction.md). This will give you an understanding of what goes on and the different topics you need to consider.
+Before upgrading, it is recommended to **read the introduction to [upgrades in Umbraco](./upgrade-introduction.md)**. This provides an understanding of what happens and the different topics you need to consider.
```

Comment:
Since we are referring to **read the introduction** in the article. There is no Introduction section as such in the article.

Can it be rephrased to "Before upgrading, it is recommended to **read the [Upgrades in Umbraco](./upgrade-introduction.md)** article. This article provides an understanding of what is involved and the different topics you need to consider while upgrading."